# Takes 1 point and returns status
geometry_msgs/Pose2D target_pos
bool disable_collisions
---
bool success
---
# No feedback